handsontable/src/dataMap/metaManager/mods/fixedColumnsProperties.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @class FixedColumnsPropertiesMod
+ */
+export class FixedColumnsPropertiesMod {
```

Comment:
I think that the Modifier should be implemented that way that it handles meta extension not only for properties used in RTL but globally, for all future needs.

This proposal prompted me to prepare a more general Mod. Can you [check it out](https://github.com/handsontable/handsontable/compare/feature/issue-8763...feature/issue-8763-property-aliasing-v2?expand=1) and say what you think about it?

Reply:
I really appreciate your proposition, thank you. I'm in doubt about the impact of what it's created. I would start of that, we have got no requirement for aliasing more options. So following the YAGNI rule, it's redundant. I'm suspecting that the aliasing is nothing we want to introduce because it complicates our public API. 

Additionally, currently, the mod has single responsibility to maintain aliasing for fixed columns within specified restrictions.
Making it more generic introduces more responsibilities to the class because the new Mod will maintain a collection of aliases and theirs own custom restrictions. I would like to have more simple (atomic) mods than one mod with complex logic.
What do you think? Would you mind accepting that?

Reply:
I see a high chance of a situation when we will stop supporting the `fixedColumnsLeft` (for instance in 2030), and no other alias added since then.

Before adding another alias we have no profit from that solution, and it will introduce another loop inside and makes code harder to understand.

The bundle size also will be larger until we introduce at least one or two additional aliases.

Would/should we like to ask @warpech for a decision? I'm going to accept his verdict.

Reply:
This will be a complex answer :) I hate to be the judge here. I think you are both right. 

I agree with everything that @Siemienik is saying. Regarding all aspects of YAGNI presented by him. @Siemienik's code is straightforward and shorter.

I totally understand @budnix arguments and agree with them, as well. The problem with YAGNI principle in Handsontable is that in the past we have struggled to refactor code "just-in-time". Very often we refactor when it is a bit too late. In the future, when there is a need for a second alias, there is a high chance that someone will duplicate `FixedColumnsPropertiesMod`, instead of choosing the general solution proposed in f9a1a49956cf147163cbcd66378eabf8a391e416 (which would be the right way).

The problem with @budnix's solution, however, is that it is generalised about something we don't even know about. For exmple:
1. The `observeTarget: false` code path is not used at all. It should be removed immediately.
2. If the next alias in the future needs to support cascading configuration, this solution will need to be changed. 
 
Wrapping up, and respecting @budnix experience with the project: I lean towards the `ExtendMetaPropertiesMod`, because when it is in place, in future it will be clearer what changes are needed and where to apply them.

Alternatively, we could go with @Siemienik's solution and add a comment with a link to f9a1a49956cf147163cbcd66378eabf8a391e416 in case someone wants to add a second "extender". 

In any case, it is great that we have unit tests in `fixedColumnsProperties.unit.js`.

Thanks to both of you for caring about the code quality.

Reply:
Ok, I merged `FixedColumnsPropertiesMod`. 
Thank you for constructive discussion 🥇